lib/jsdom/level2/events.js

```diff
@@ -9,26 +9,7 @@ var core = require("../level1/core"),
     defineSetter = utils.defineSetter,
     inheritFrom = utils.inheritFrom;
 
-core.EventException = function() {
-    if (arguments.length > 0) {
-        this._code = arguments[0];
-    } else {
-        this._code = 0;
-    }
-    if (arguments.length > 1) {
-        this._message = arguments[1];
-    } else {
-        this._message = "Unspecified event type";
-    }
-    Error.call(this, this._message);
-    if (Error.captureStackTrace) {
-        Error.captureStackTrace(this, core.EventException);
-    }
-};
-inheritFrom(Error, core.EventException, {
-  UNSPECIFIED_EVENT_TYPE_ERR : 0,
-  get code() { return this._code;}
-});
+core.EventException = require("../events/EventException").EventException;
```

Comment:
`core` is `dom` which is how we currently build our window, could move that out while we're at it already but we don't have an infrastructure for that currently. Maybe build some kind of imports-define object and walk through it in the Window constructor?